src/tax-lot/tax-lot.controller.ts

```diff
@@ -36,44 +36,42 @@ export class TaxLotController {
   constructor(private readonly taxLotService: TaxLotService) {}
 
   @Get("/:bbl")
-  @UsePipes(new ZodValidationPipe(getTaxLotByBblPathParamsSchema))
-  async findDetailsByBbl(@Param() params: GetTaxLotByBblPathParams) {
-    return this.taxLotService.findTaxLotByBbl(params.bbl);
+  @UsePipes(new ZodValidationPipe(findTaxLotByBblPathParamsSchema))
+  async findDetailsByBbl(@Param() params: FindTaxLotByBblPathParams) {
+    return this.taxLotService.findByBbl(params.bbl);
   }
 
   @Get("/:bbl/geojson")
-  @UsePipes(new ZodValidationPipe(getTaxLotGeoJsonByBblPathParamsSchema))
+  @UsePipes(new ZodValidationPipe(findTaxLotGeoJsonByBblPathParamsSchema))
   async findTaxLotByBblGeoJson(
-    @Param() params: GetTaxLotGeoJsonByBblPathParams,
+    @Param() params: FindTaxLotGeoJsonByBblPathParams,
   ) {
-    return this.taxLotService.findTaxLotByBblGeoJson(params.bbl);
+    return this.taxLotService.findByBblGeoJson(params.bbl);
   }
 
   @Get("/:bbl/zoning-districts")
   @UsePipes(
-    new ZodValidationPipe(getZoningDistrictsByTaxLotBblPathParamsSchema),
+    new ZodValidationPipe(findZoningDistrictsByTaxLotBblPathParamsSchema),
   )
   async findZoningDistrictByTaxLotBbl(
```

Comment:
1. The original method name misrepresents the resulting zoning districts as singular when it is actually plural. 
2. 'TaxLot' is already implied because we are in the tax lot domain.

This applies to corresponding service and repository methods, as well

```suggestion
  async findZoningDistrictsByBbl(
```